package.json

```diff
@@ -9,7 +9,8 @@
     "bootstrap": "lerna bootstrap",
     "prepare": "if test \"$NODE_ENV\" != \"production\" && test \"$CI\" != \"true\" ; then husky install ; fi",
     "prepack": "lerna run --stream prepack",
-    "dev": "lerna run --stream prepack -- --incremental && lerna --ignore=@logto/integration-test run --parallel dev",
+    "dev:watch": "lerna --ignore=@logto/integration-test run --parallel dev",
```

Comment:
But why it doesn't use `watch` in this command?

Reply:
removed this script name